services/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const SECONDS_PER_MINUTE = 60;
+export const SECONDS_PER_HOUR = 3600;
```

Comment:
Please move this file to `constant` directory.

Reply:
Done in https://github.com/rango-exchange/explorer/pull/32/commits/65f30b5f9f842a4c6ef38892cda4558bb0f1b576